src/main.js

```diff
@@ -0,0 +1,15 @@
+$(function() {
+  $('form').on('submit', function (event) {
+    event.preventDefault()
+    var id = $('titleSearch').val()
+    var urlMovie = 'http://img.omdbapi.com/?t=' + id
```

Comment:
looks perfect. We might need to adjust for multi-word titles, since the url won't recognize spaces.